src/systems/track_controller/TrackController.hh

```diff
@@ -82,6 +82,14 @@ namespace systems
   ///   rotation command will only act for the given number of seconds and the
   ///   track will be stopped if no command arrives before this timeout.
   ///
+  /// `<odometry_topic>` The topic on which the track odometry (i.e. position
+  ///   and instantaneous velocity) is published. This can be used e.g. to
+  ///   simulate a conveyor with encoder feedback.
+  ///   Defaults to `/model/${model_name}/link/${link_name}/odometry`.
+  /// 
```

Comment:
CI is not happy with this line because the line ends in white space.
```suggestion
  ///
```

Reply:
https://github.com/gazebosim/gz-sim/pull/2021/commits/a04faa61c2c7bbc4a11a421223120e7973d6529b